examples/locality-load-balancing/Dockerfile-client

```diff
@@ -0,0 +1,11 @@
+FROM envoyproxy/envoy-dev:latest
+RUN apt-get update && apt-get install -y bash curl jq python3 vim iputils-ping
+
+COPY ./envoy-client.yaml /etc/envoy.yaml
```

Comment:
can we rename this file `-client` -> `-proxy` - likewise the Dockerfile

i think naming it client is confusing to the setup as its actually the load balancing proxy (even if it also has the client script)

perhaps it would also help to move the `client.py` script to its own container

Reply:
> perhaps it would also help to move the client.py script to its own container

I don't understand it. Can you clarify in which container should we move `client.py` to?